frontend/src/routes/(admin)/settings/aliases/index.tsx

```diff
@@ -3,7 +3,8 @@ import { loader$ } from '@builder.io/qwik-city'
 import { checkAuth } from '~/utils/checkAuth'
 
 export const loader = loader$(async ({ request, platform, redirect }) => {
-	const isAuthorized = await checkAuth(request, platform)
+	const jwt = request.headers.get('Cf-Access-Jwt-Assertion') || ''
```

Comment:
this should be using CF_Authorization in cookies as the route isn't running Access

Reply:
ah ok thanks I thought so too but the old implementation was using the header, so I figured that maybe I was wrong 😓 👍